### ISSUE TITLE
bounds behaviour doesn't match docstring

### ISSUE THREAD

**Issue by aburrell** [pysat/pysat#259]
**Describe the bug**
The docstring for the Instrument object attribute `bounds` does not behave as described.  In the docstring, it is specified that bounds may either be a tuple of datetime objects or a tuple of lists of datetime objects.  However, when setting bounds, only a tuple of lists is returned.

**To Reproduce**
```
import datetime as dt
import pysat

bounds = (dt.datetime(2014, 1, 1), dt.datetime(2014, 1, 3))
omni = pysat.Instrument('omni', 'hro', tag='1min', clean_level='clean')
omni.bounds = bounds
print(bounds)
print(omni.bounds)
```

Returns
```
(datetime.datetime(2014, 1, 1, 0, 0), datetime.datetime(2014, 1, 3, 0, 0))
([datetime.datetime(2014, 1, 1, 0, 0)], [datetime.datetime(2014, 1, 3, 0, 0)])
```

**Expected behavior**
The documentation should be updated to express the real behaviour, or the behaviour should be altered to reflect the docstring.

**Desktop (please complete the following information):**
 - OS: OS X Sierra
- Python 2.7

**Comment by rstoneback** [comment]
Thanks for catching this. I believe the doc string should be updated to note that it accepts multiple forms as inputs but the format bounds is stored in remains consistent.

Feature intended to make it easy to set simple start and end dates as well as account for multiple start/stop times.

Cheers,
Russell

On Aug 13, 2019, at 10:08 AM, Angeline Burrell <notifications@github.com<mailto:notifications@github.com>> wrote:


Describe the bug
The docstring for the Instrument object attribute bounds does not behave as described. In the docstring, it is specified that bounds may either be a tuple of datetime objects or a tuple of lists of datetime objects. However, when setting bounds, only a tuple of lists is returned.

To Reproduce

import datetime as dt
import pysat

bounds = (dt.datetime(2014, 1, 1), dt.datetime(2014, 1, 3))
omni = pysat.Instrument('omni', 'hro', tag='1min', clean_level='clean')
omni.bounds = bounds
print(bounds)
print(omni.bounds)


Returns

(datetime.datetime(2014, 1, 1, 0, 0), datetime.datetime(2014, 1, 3, 0, 0))
([datetime.datetime(2014, 1, 1, 0, 0)], [datetime.datetime(2014, 1, 3, 0, 0)])


Expected behavior
The documentation should be updated to express the real behaviour, or the behaviour should be altered to reflect the docstring.

Desktop (please complete the following information):

  *   OS: OS X Sierra
  *   Python 2.7

**Comment by jklenzing** [comment]
Closed with merge of #290.
